release-notes/6.0/install-linux.md

```diff
@@ -0,0 +1,78 @@
+# .NET 6 installation instructions for Linux
+
+The following instructions demonstrate installing .NET 6 on Linux. These instructions augment the more general [.NET install instructions](install.md), including installing with `.tar.gz` files, that work on multiple operating systems.
+
+[Linux package dependencies](linux-packages.md) describes the set of packages required to run .NET on Linux.
+
+## Distributions
+
+The following distributions are available for Linux:
+
+- .NET SDK: includes tools for building and testing applications, and includes the runtime distributions that follow.
+- .NET Runtime: includes the .NET runtime and libraries, enabling running console applications.
+- ASP.NET Core Runtime: includes the .NET and ASP.NET Core runtimes, enabling running console, and web applications.
+
+You are recommended to install the .NET SDK to develop and build applications, and to install one of the runtimes packages (like ASP.NET Core) to exclusively run applications.
```

Comment:
Wording? Perhaps: "We recommend you install..." Also same wording for the 3 other variants of the install*.md docs.

Reply:
Good call.